docs/spec/v1beta2/helmrepositories.md

```diff
@@ -107,9 +177,11 @@ change to the spec), this is handled instantly outside the interval window.
 
 ### URL
 
-`.spec.url` is a required field that specifies the HTTP/S address of the Helm
-repository. For Helm repositories which require authentication, see
-[Secret reference](#secret-reference).
+`.spec.url` is a required field that specifies the HTTP/S or OCI address of a
+Helm repository depending on the [type of the HelmRepository object](#type).
```

Comment:
```suggestion
`.spec.url` is a required field that depending on the [type of the HelmRepository object](#type)
specifies the HTTP/S or OCI address of a Helm repository. 
```

It was late yesterday night, but looking at it now, I think this order would work better (and also: is better English).